go.mod

```diff
@@ -6,7 +6,7 @@ require (
 	github.com/Azure/azure-sdk-for-go v61.6.0+incompatible
 	github.com/Azure/go-autorest/autorest v0.11.21
 	github.com/Azure/go-autorest/autorest/azure/auth v0.5.7
-	github.com/cloudquery/cq-provider-sdk v0.10.8
+	github.com/cloudquery/cq-provider-sdk v0.10.9
```

Comment:
A rebase should clear these changes from the PR I believe

Reply:
Updated